### PR TITLE
feat(launch): revert chain launch message

### PR DESCRIPTION
**Pull request by lubtd**
`MsgRevertChainLaunch` allows to set the `LaunchTriggered` on-chain value back to false if something goes wrong with the chain. So it can be relaunched in the future.

This message can be sent after the launch time of the chain + a delay (1 hour)

### CODE REVIEW

File: x/launch/keeper/msg_server_revert_launch.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	// Check sender is the coordinator of the chain
+	coordinatorID, found := k.profileKeeper.CoordinatorIDFromAddress(ctx, msg.Coordinator)
+	if !found {
+		return nil, sdkerrors.Wrap(profiletypes.ErrCoordAddressNotFound, msg.Coordinator)
+	}
+	if chain.CoordinatorID != coordinatorID {
+		return nil, sdkerrors.Wrap(profiletypes.ErrCoordInvalid, fmt.Sprintf(
+			"coordinator of the chain is %v",
+			chain.CoordinatorID,
+		))
```
[the remark this answers is not in the document]
Comment (lubtd) [reply x/launch/keeper/msg_server_revert_launch.go:31]: This line is actually pretty long when you use a vertical screen.
I would prefer to use multiline when 120 chars is reached for other people using a vertical screen

[the remark this answers is not in the document]
Comment (Pantani) [reply x/launch/keeper/msg_server_revert_launch.go:31]: Sure!! I agree, but I think to use the `sdkerrors.Wrapf` instead `sdkerrors.Wrap` with a `Sprintf` inside it's more clear:

```suggestion
		return nil, sdkerrors.Wrapf(
			profiletypes.ErrCoordInvalid, 
			"coordinator of the chain is %v", 
			chain.CoordinatorID,
		)
```

[the remark this answers is not in the document]
Comment (lubtd) [reply x/launch/keeper/msg_server_revert_launch.go:31]: Oh, I haven't realized it was the point of the suggestion🙄

**Review by Pantani — DISMISSED** [review]

**Review by ilgooz — APPROVED** [review]

**Review by Pantani — APPROVED** [review]
🦾

**Pull request merged**